app/src/components/base/DetailCard/DetailCard.tsx

```diff
@@ -10,8 +11,8 @@ export const DetailCard: React.FC<DetailCardProps> = ({
   footContent,
 }) => {
   return (
-    <InfoCard>
-      {!!headContent && <HeadContentWrapper>{headContent}</HeadContentWrapper>}
+    <InfoCardSection>
+      {!!headContent && <Card.Header>{headContent}</Card.Header>}
```

Comment:
```suggestion
      {!!headContent && <InfoCardSection.Header>{headContent}</InfoCardSection.Header>}
```

Reply:
Yeah, that was the first thing I tried.
<img width="1262" alt="Screenshot 2023-05-05 at 6 00 53 PM" src="https://user-images.githubusercontent.com/88854201/236575802-4d56ce04-8e1c-416e-aec0-f7ad4c28f14d.png">